resources/dicts/thoughts/alive/general.json

```
@@ -538,7 +538,7 @@
             "Has been imitating the Clan leader's behaviour recently",
             "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a shoo-in for deputy",
             "Wonders if the Clan leader talks about {PRONOUN/m_c/object} when {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} not around",
-            "Tells the Clan leader {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
+            "Tells lead_name {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
```

not sure why we're utilizing lead_name here when the r_c cat is already constrained to being the leader?  It would be more consistent, imo, if we just used r_c.
```suggestion
            "Tells r_c {PRONOUN/r_c/subject}{VERB/r_c/'ve/'s} been doing a wonderful job leading"
```